tests/bluetooth/tester/src/btp_vcp.c

```diff
@@ -0,0 +1,553 @@
+		vol_up();
+		break;
+	case VCS_VOL_DOWN:
+		vol_down();
+		break;
+	case VCS_MUTE:
+		mute();
+		break;
+	case VCS_UNMUTE:
+		unmute();
+		break;
+	default:
+		tester_rsp(BTP_SERVICE_ID_VCS, opcode, index,
+			   BTP_STATUS_UNKNOWN_CMD);
+		break;
+	}
+}
+
+/* Audio Input Control Service */
+static void aics_supported_commands(uint8_t *data, uint16_t len)
+{
+	struct net_buf_simple *buf = NET_BUF_SIMPLE(BTP_DATA_MAX_SIZE);
+
+	net_buf_simple_init(buf, 0);
+	net_buf_simple_add_u8(buf, AICS_READ_SUPPORTED_COMMANDS);
+	net_buf_simple_add_u8(buf, AICS_SET_GAIN);
+	net_buf_simple_add_u8(buf, AICS_MUTE);
+	net_buf_simple_add_u8(buf, AICS_UNMUTE);
+	net_buf_simple_add_u8(buf, AICS_MAN_GAIN);
+	net_buf_simple_add_u8(buf, AICS_AUTO_GAIN);
+	net_buf_simple_add_u8(buf, AICS_DESCRIPTION);
+
+	tester_send(BTP_SERVICE_ID_AICS, AICS_READ_SUPPORTED_COMMANDS,
+		    CONTROLLER_INDEX, buf->data, buf->len);
+}
+
+static void aics_state_cb(struct bt_aics *inst, int err, int8_t gain,
+			  uint8_t mute, uint8_t mode)
+{
+	LOG_DBG("AICS state callback");
+}
+
+static void aics_gain_setting_cb(struct bt_aics *inst, int err, uint8_t units,
+				 int8_t minimum, int8_t maximum)
+{
+	LOG_DBG("AICS gain setting callback");
+}
+
+static void aics_input_type_cb(struct bt_aics *inst, int err,
+			       uint8_t input_type)
+{
+	LOG_DBG("AICS input type callback");
+}
+
+static void aics_status_cb(struct bt_aics *inst, int err, bool active)
+{
+	LOG_DBG("AICS status callback");
+}
+
+static void aics_description_cb(struct bt_aics *inst, int err,
+				char *description)
+{
+	LOG_DBG("AICS description callback");
+}
+
+static struct bt_aics_cb aics_cb = {
+	.state = aics_state_cb,
+	.gain_setting = aics_gain_setting_cb,
+	.type = aics_input_type_cb,
+	.status = aics_status_cb,
+	.description = aics_description_cb
+};
+
+void aics_set_gain(uint8_t *data, uint8_t len)
+{
+	const struct aics_set_gain *cmd = (void *)data;
+	int8_t gain;
+	int err;
+
+	gain = sys_le16_to_cpu(cmd->gain);
+
+	LOG_DBG("AICS set gain 0x%02x", gain);
+
+	for (int i = 0; i < CONFIG_BT_VCP_VOL_REND_AICS_INSTANCE_COUNT; i++) {
+		err = bt_aics_gain_set(included.aics[0], gain);
+		if (err) {
+			tester_rsp(BTP_SERVICE_ID_AICS, AICS_SET_GAIN,
+				   CONTROLLER_INDEX, BTP_STATUS_FAILED);
+			return;
+		}
+	}
+	tester_rsp(BTP_SERVICE_ID_AICS, AICS_SET_GAIN, CONTROLLER_INDEX,
+		   BTP_STATUS_SUCCESS);
+}
+
+void aics_mute(void)
+{
+	int err;
+
+	LOG_DBG("AICS mute");
+
+	for (int i = 0; i < CONFIG_BT_VCP_VOL_REND_AICS_INSTANCE_COUNT; i++) {
+		err = bt_aics_mute(included.aics[i]);
+		if (err) {
+			tester_rsp(BTP_SERVICE_ID_AICS, AICS_MUTE,
+				   CONTROLLER_INDEX, BTP_STATUS_FAILED);
+			return;
+		}
+	}
+	tester_rsp(BTP_SERVICE_ID_AICS, AICS_MUTE, CONTROLLER_INDEX,
+		   BTP_STATUS_SUCCESS);
+}
+
+void aics_unmute(void)
+{
+	int err;
+
+	LOG_DBG("AICS unmute");
+
+	for (int i = 0; i < CONFIG_BT_VCP_VOL_REND_AICS_INSTANCE_COUNT; i++) {
+		err = bt_aics_unmute(included.aics[i]);
+		if (err) {
```

Comment:
In all these loops: Suggest to move the declaration `err` to the `for` loop scope to minimize the scope

Reply:
Done

Reply:
CI warns me about the blank lines, I've added them - but don't know if this up to standards?